docs/installation_guide.md

```diff
@@ -1265,146 +1345,164 @@ Check your python requests module version.
+11. The view should automatically change to the _Weather Example Mashup_ workspace and widgets should appear in it:
 
     <img src="../images/installation_guide/weather_dashboard.png" srcset="../images/installation_guide/weather_dashboard.png 2x" alt="Final weather dashboard">
 
-12. Select the pin tool in the *Web Map Service* widget clicking the appropriated button as shown in the image.
-13. And click the desired location. The *Weather Widget Example* should update the forecast info:
+12. Select the pin tool in the _Web Map Service_ widget clicking the appropriated button as shown in the image.
+13. And click the desired location. The _Weather Widget Example_ should update the forecast info:
 
     <img src="../images/installation_guide/example_usage.png" srcset="../images/installation_guide/example_usage.png 2x" alt="Example usage">
 
-By performing this sequence of steps, you will check that the WireCloud Mashup
-platform is running and correctly deployed, and its database has been properly
-set up and populated.
+By performing this sequence of steps, you will check that the WireCloud Mashup platform is running and correctly
+deployed, and its database has been properly set up and populated.
 
 ### List of Running Processes
 
-We need to check that the Apache web server and the Postgres database are running. WireCloud uses a python interpreter, but it will not be listed as it runs embedded into apache2. If we execute the following command:
+We need to check that the Apache web server and the Postgres database are running. WireCloud uses a python interpreter,
+but it will not be listed as it runs embedded into apache2. If we execute the following command:
 
-    ps -ewF | grep 'apache2\|postgres' | grep -v grep
+```bash
+ps -ewF | grep 'apache2\|postgres' | grep -v grep
+```
 
 It should show something similar to the following:
 
-    $ ps -ewF | grep 'apache2\|postgres' | grep -v grep
-    postgres  1631     1  0 25212  9452   0 Jul03 ?        00:00:19 /usr/lib/postgresql/9.1/bin/postgres -D /var/lib/postgresql/9.1/main -c config_file=/etc/postgresql/9.1/main/postgresql.conf
-    postgres  1702  1631  0 25208  3784   0 Jul03 ?        00:00:47 postgres: writer process
-    postgres  1703  1631  0 25208  1452   0 Jul03 ?        00:00:39 postgres: wal writer process
-    postgres  1704  1631  0 25462  2964   0 Jul03 ?        00:00:16 postgres: autovacuum launcher process
-    postgres  1705  1631  0 17370  1660   0 Jul03 ?        00:00:18 postgres: stats collector process
-    root      3811     1  0 50067 10848   0 13:13 ?        00:00:00 /usr/sbin/apache2 -k start
-    www-data  3818  3811  0 68663 39820   0 13:13 ?        00:00:00 /usr/sbin/apache2 -k start
-    www-data  3819  3811  0 68687 39448   0 13:13 ?        00:00:00 /usr/sbin/apache2 -k start
-    www-data  3822  3811  0 68901 40160   0 13:13 ?        00:00:00 /usr/sbin/apache2 -k start
+```bash
```

Comment:
don't split, use the `console` language.

Reply:
Unfortunately your default choices here are `bash` or `text` - `console` isn't supported on readthedocs.